com.unity.sg2/Editor/GraphUI/DataModel/Blackboard/GraphDataVariableDeclarationModel.cs

```diff
@@ -49,7 +49,7 @@ public override void CreateInitializationValue()
                 InitializationModel = GraphModel.Stencil.CreateConstantValue(DataType);
                 if (InitializationModel is BaseShaderGraphConstant cldsConstant)
                 {
-                    cldsConstant.Initialize(shaderGraphModel.GraphHandler, contextNodeName, graphDataName);
+                    cldsConstant.Initialize(shaderGraphModel, contextNodeName, graphDataName);
```

Comment:
For source of truth assurances, the constants will fetch the handler through the ShaderGraphModel.